tests/elements-regression/src/pages/editor-page.js

```diff
@@ -1,6 +1,12 @@
-const BasePage = require( './base-page.js' );
+export default class EditorPage {
```

Comment:
Can we use the same EditorPage that we use in the sanity tests?

Reply:
fixed